nvd3/linePlusBarChart.py

```diff
@@ -61,12 +61,12 @@ class linePlusBarChart(TemplateMixin, NVD3Chart):
                 chart.margin({top: 30, right: 60, bottom: 20, left: 60});
                 var datum = data_linePlusBarChart;
 
-                    chart.y2Axis
-                        .tickFormat(function(d) { return d3.format(',0.3f')(d) });
-                    chart.xAxis
-                        .tickFormat(function(d) { return d3.time.format('%d %b %Y')(new Date(parseInt(d))) });
-                    chart.y1Axis
-                        .tickFormat(function(d) { return d3.format(',f')(d) });
+                        chart.y2Axis
+                            .tickFormat(function(d) { return d3.format(',0.3f')(d) });
+                        chart.xAxis
+                            .tickFormat(function(d) { return d3.time.format('%d %b %Y')(new Date(parseInt(d))) });
+                        chart.y1Axis
+                            .tickFormat(function(d) { return d3.format(',f')(d) });
```

Comment:
Reply: "is there a gain to reindent ?"
Err... I just add the self.create_x_axis and self.create_y_axis in "linePlusBarChart.py", seriously.

At 2015-08-16 21:30:49, "Areski Belaid" <notifications@github.com> wrote:


In nvd3/linePlusBarChart.py:

> @@ -61,12 +61,12 @@ class linePlusBarChart(TemplateMixin, NVD3Chart):
>                  chart.margin({top: 30, right: 60, bottom: 20, left: 60});
>                  var datum = data_linePlusBarChart;
>  
> -                    chart.y2Axis
> -                        .tickFormat(function(d) { return d3.format(',0.3f')(d) });
> -                    chart.xAxis
> -                        .tickFormat(function(d) { return d3.time.format('%d %b %Y')(new Date(parseInt(d))) });
> -                    chart.y1Axis
> -                        .tickFormat(function(d) { return d3.format(',f')(d) });
> +                        chart.y2Axis
> +                            .tickFormat(function(d) { return d3.format(',0.3f')(d) });
> +                        chart.xAxis
> +                            .tickFormat(function(d) { return d3.time.format('%d %b %Y')(new Date(parseInt(d))) });
> +                        chart.y1Axis
> +                            .tickFormat(function(d) { return d3.format(',f')(d) });


is there a gain to reindent ?

—
Reply to this email directly or view it on GitHub.